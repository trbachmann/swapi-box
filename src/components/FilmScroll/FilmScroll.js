import React from 'react';

export const FilmScroll = ({ film }) => {
  return(
    <div className="film-scroll">
      <p>{film.opening_crawl}</p>
      <p>{film.title}</p>
      <p>{film.release_date}</p>
      <p>Choose a Category</p>
    </div>
  )
}
