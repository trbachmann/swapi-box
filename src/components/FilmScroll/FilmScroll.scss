.film-scroll {
  color: #FFE300;
  width: 22%;
  margin: 2%;
  font-size: 1.1em;
}