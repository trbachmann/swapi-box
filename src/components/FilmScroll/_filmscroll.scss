.film-scroll {
  color: $yellow;
  margin: 2%;
  font-size: 1.1em;
}