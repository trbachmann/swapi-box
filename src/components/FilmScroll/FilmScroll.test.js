import React from 'react';
import { FilmScroll } from './FilmScroll';
import { shallow } from 'enzyme';

describe('FilmScroll', () => {
  const film = {
    opening_crawl: 'Turmoil has engulfed the Galactic Republic.',
    title: 'The Phantom Menace',
    release_date: '1999-05-09'
  };

  it('should match the snapshot with all data passed in', () => {
    const wrapper = shallow(
      <FilmScroll film={film} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});