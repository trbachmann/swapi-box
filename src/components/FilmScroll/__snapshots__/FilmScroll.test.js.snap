// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmScroll should match the snapshot with all data passed in 1`] = `
<div
  className="film-scroll"
>
  <p>
    Turmoil has engulfed the Galactic Republic.
  </p>
  <p>
    The Phantom Menace
  </p>
  <p>
    1999-05-09
  </p>
  <p>
    Choose a Category
  </p>
</div>
`;
